### PR TITLE
Added Grids Component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Grids Component

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can merge

**Review by Hamdrive — COMMENTED** [review]
Apart from cleaning up the DS file, everything else looks good!
Can be merged

File: styles/docs.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
 @import url(/styles/modal.css);
 @import url(/styles/ratings.css);
 @import url(/styles/snackbar.css);
+@import url(/styles/grid.css);
```
Comment (MaruthiPothuganti) [inline styles/docs.css:15]: for side menu, you can use

> you can also try position: absolute and left:0, top:0
> making the parent as relative to fix that nav aside

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good.

File: styles/docs.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/docs.css:15]: Thanks for the review, changes made

**Pull request merged**